external/loaders/loaders/batches/_batch.py

```diff
@@ -80,6 +81,7 @@ class BatchesFromMapperConfig(BatchesLoader):
     shuffle_samples: bool = False
     data_transforms: Optional[Sequence[Mapping]] = None
     ptop: float = vcm.calc.thermo.constants.TOA_PRESSURE
+    catalog_path: str = vcm.catalog.catalog_path
```

Comment:
Could you also add an entry to the docstring describing this argument?  This might be a good place to mention the minimal required entries for such a catalog, namely `"grid/{res}"`, `"landseamask/{res}"`, and `"wind_rotation/{res}"` for a given resolution.